src/adapters/config.ts

```diff
@@ -370,6 +370,15 @@ class Config {
     }
   }
 
+  public async addToken(body: string) {
+    const resp = await fetch(`${API_BASE_URL}/configs/custom-tokens`, {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: body,
+    })
+    return resp.status
+  }
```

Comment:
please check if the `resp.ok` to see if request failed or not
- if failed throw error
- else you should return `resp.json()` -> convert the response to json, normally POST request return response payload